### PR TITLE
[docs/gettingstarted] add interactive debug instructions

### PR DESCRIPTION
**Pull request by arthurzenika**

### CODE REVIEW

File: docs/gettingstarted.md [hunk 1]
```diff
@@ -269,6 +269,19 @@ With either of these, test-kitchen will do the following
...
+If your `converge` or `verify` step is failing, by default `kitchen` will keep your VM running, so you can login using 
+ssh and debug things from there. To run the `state.apply` that converge is doing, run the following : 
+
+    kitchen login
+    sudo salt-call --config-dir=etc/salt/ state.apply
+
+If you are using the `minion_id` argument run : 
+
+    kitchen login
+    sudo salt-call --config-dir=etc/salt/ --id=salt-minion-id state.apply
```
Comment (arthurzenika) [inline docs/gettingstarted.md]:
```suggestion
    sudo salt-call --config-dir=/tmp/kitchen/etc/salt/ --log-level=info --id=salt-minion-id state.apply
```

File: docs/gettingstarted.md [hunk 2]
```diff
@@ -269,6 +269,19 @@ With either of these, test-kitchen will do the following
 
 If `kitchen test` was run, then the `kitchen list` command will show all the instances and what the result of their last command was.
 
+## Run test interactivaly ##
+
+If your `converge` or `verify` step is failing, by default `kitchen` will keep your VM running, so you can login using 
+ssh and debug things from there. To run the `state.apply` that converge is doing, run the following : 
+
+    kitchen login
+    sudo salt-call --config-dir=etc/salt/ state.apply
```
Comment (arthurzenika) [inline docs/gettingstarted.md]:
```suggestion
    sudo salt-call --config-dir=/tmp/kitchen/etc/salt/ --log-level=info state.apply 
```

**Review by s0undt3ch — APPROVED** [review]

File: docs/gettingstarted.md [hunk 3]
```diff
@@ -269,6 +269,19 @@ With either of these, test-kitchen will do the following
 
 If `kitchen test` was run, then the `kitchen list` command will show all the instances and what the result of their last command was.
 
+## Run test interactivaly ##
+
+If your `converge` or `verify` step is failing, by default `kitchen` will keep your VM running, so you can login using 
+ssh and debug things from there. To run the `state.apply` that converge is doing, run the following : 
+
+    kitchen login
+    sudo salt-call --config-dir=/tmp/kitchen/etc/salt/ --log-level=info state.apply
```
Comment (daks) [inline docs/gettingstarted.md]: you can even use `--log-level=debug`

File: docs/gettingstarted.md [hunk 4]
```diff
@@ -269,6 +269,19 @@ With either of these, test-kitchen will do the following
...
+If your `converge` or `verify` step is failing, by default `kitchen` will keep your VM running, so you can login using 
+ssh and debug things from there. To run the `state.apply` that converge is doing, run the following : 
+
+    kitchen login
+    sudo salt-call --config-dir=/tmp/kitchen/etc/salt/ --log-level=info state.apply 
+
+If you are using the `minion_id` argument run : 
+
+    kitchen login
+    sudo salt-call --config-dir=/tmp/kitchen/etc/salt/ --log-level=info --id=salt-minion-id state.apply
```
Comment (daks) [inline docs/gettingstarted.md]: here too

**Review by daks — APPROVED** [review]

File: docs/gettingstarted.md (hunk 3 above)
Comment (arthurzenika) [inline docs/gettingstarted.md]:
```suggestion
    sudo salt-call --config-dir=/tmp/kitchen/etc/salt/ --log-level=debug state.apply 
```

File: docs/gettingstarted.md (hunk 4 above)
Comment (arthurzenika) [inline docs/gettingstarted.md]:
```suggestion
    sudo salt-call --config-dir=/tmp/kitchen/etc/salt/ --log-level=debug --id=salt-minion-id state.apply
```

File: docs/gettingstarted.md (hunk 1 above)
Comment (n-rodriguez) [reply docs/gettingstarted.md]: Hi there! `bin/kitchen` is preferred over `kitchen`.

File: docs/gettingstarted.md (hunk 2 above)
Comment (n-rodriguez) [reply docs/gettingstarted.md]: Same here.

**Comment by arthurzenika** [comment]
@n-rodriguez I don't see the rest of the documentation talking about `bin/kitchen` are you sure that's a good idea ? In most cases people will install kitchen system wide I think.

**Comment by n-rodriguez** [comment]
@arthurlogilab  sorry, I misread the PR, I thought we were on a formula ^^

**Comment by arthurzenika** [comment]
@n-rodriguez no problem, thanks for reading the PR !

**Comment by daks** [comment]
I think this PR has been forgotten and can be merged, what do you think @s0undt3ch @n-rodriguez?

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**